server/server.go

```diff
@@ -489,7 +499,7 @@ func (server HTTPServer) fetchRuleContent(rule types.RuleOnReport) (proxy_types.
 		Internal:        ruleWithContent.Internal,
 	}
 
-	return parsedRule, true
+	return parsedRule, true, true
```

Comment:
Why 3rd one is true? According to the code below it is "isError", but it doesn't look like any error happened here.

Reply:
also, having 2 return values indicating errors by opposite values(successful is true = "ok", isError is false = "ok") is kinda confusing

Reply:
Fixed variable name.